pkg/autoscaler/autoscaler.go

```diff
@@ -173,14 +173,15 @@ var _ = Describe("Autoscaler should", framework.LabelAutoscaler, Serial, func()
 
 			By("Creating ClusterAutoscaler")
 			clusterAutoscaler = clusterAutoscalerResource(100)
-			Expect(client.Create(ctx, clusterAutoscaler)).Should(Succeed())
+			err = client.Create(ctx, clusterAutoscaler)
+			Expect(err).Should(Succeed(), "Failed to create ClusterAutoscaler resource, error: %v", err)
```

Comment:
We probably don't need to print the error into these messages, since the gomega library already prints them for us, can probably keep this inline, WDYT?